client/src/App.jsx

```diff
@@ -1,35 +1,34 @@
 import { useState, useEffect } from 'react';
+//'switch' has been replaced with 'routes'
```

Comment:
```suggestion
//'Switch' has been replaced with 'Routes'
```
Even in the comments, paying attention to the capitalization will help you in the future.

Reply:
Okay, thank you for bringing this to my attention.